doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -1990,30 +1990,76 @@ Response:
+update operation, and the registrations allow to forward requests, 
+implementations must forward a request to such registered context provider so that it can provide
+the requested data or update it.
+The forwarding algorithm and precise implementation details are work to be completed
+in next versions of this specification. (*TODO*)
 
 A context registration is represented by a JSON object with the following fields:
 
-+ `id`       : Unique identifier assigned to the registration. Automatically generated at creation
-  time.
-+ `subject`  : Object that describes the subject of the registration.
-+ `callback` : URL which denotes the end-point of the registered service. It is optional, as it can
-  be undefined under certain conditions, for instance, transient unavailability of the provider.
-+ `metadata` : An optional JSON object which allows to associate extra metadata properties to the
-  registration.
-  It must follow the same representation conventions as context attribute metadata
-  (See JSON Attribute Representation). 
-+ `duration` : Duration of the registration in ISO8601 format. Default duration is infinite.
-
-A `subject` contains the following subfields:
++ `id` : Unique identifier assigned to the registration. Automatically generated at creation time.
++ `description` : Description given to this registration. Optional. 
++ `provider` : Object that describes the context source registered. Mandatory. 
++ `dataProvided` : Object that describes the data provided by this source. Mandatory. 
++ `status`: Enumerated field which captures the current status of this registration:
+Either `active` (for active registrations) or `inactive` (for inactive registrations).
+  If this field is not provided at registration creation time, new registrations are created with
+  the `active` status, which may be changed by clients afterwards. For expired registrations, this
+  attribute is set to `expired` (no matter if the client updates it to `active`/`inactive`).
+  Also, for registrations experiencing problems with forwarding operations, the status is set to `failed`.
+  As soon as the forwarding operations start working again, the status is changed back to `active`.
++ `expires` : Registration expiration date in ISO8601 format. Permanent registrations must omit this field.
++ `forwardingInformation`: Information related to the forwarding operations made against the provider.
+Automatically provided by the implementation. 
+
+The `provider` field contains the following subfields:
++ `http` : It is used to convey parameters for providers which deliver information through the HTTP protocol.
+(Only protocol supported nowadays).
+It must contain a subfield named `url` with the URL that serves as the endpoint that offers the providing interface.
+The endpooint must *not* include the protocol
+specific part (for instance `/v2/entities`). 
++ `legacy` : `true` if this provider only supports NGSIv1 protocol. `false` if this provider supports NGSIv2.
```

Comment:
I'd move this bullet outside the .apib, to the Orion .md. In fact, I'd suggest to add 2 sections similar to this ones (and in each one in the corresponding .md document):

https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/user/v1_v2_coexistence.md#ngsiv1-notification-with-ngsiv2-subscriptions
https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/user/ngsiv2_implementation_notes.md#legacy-attribute-format-in-notifications

Reply:
that could be part of a follow-up PR once this feature is implemented

removed at 78ce71dfa1b2c6952018ede3a9ef640c65abc1dd